ui/app/templates/components/list-accordion.hbs

```diff
@@ -0,0 +1,10 @@
+{{#each decoratedSource as |item|}}
+  {{yield (hash
+    head=(component "list-accordion/accordion-head"
+      isOpen=item.isOpen
+      onOpen=(action "open" item)
+      onClose=(action "close" item))
```

Comment:
Could use `(action (mut item.isOpen) true)` and `(action (mut item.isOpen) false)` here instead, I think.

Reply:
Good call. I always forget about `mut`.